react/components/ProductAssemblyOptions/ProductAssemblyOptionItemName.tsx

```diff
@@ -0,0 +1,9 @@
+import React, { FC } from 'react'
+import { useProductAssemblyItem } from '../ProductAssemblyContext'
+
+const Name: FC = () => {
+  const { item } = useProductAssemblyItem()
+  return <span className="t-heading-5 c-on-base">{item.name}</span>
```

Comment:
No CSS handle?

Reply:
This is a first version of something we need to get released, I think it would be smarter to leave somethings for later